GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -36,21 +36,29 @@ class ItemViewControllerSpec: QuickSpec {
                 it("sets the data source to an empty list") {
                     
                     controller.viewDidLoad()
-                    
+                    // Discuss this with @eric re: should we have declaration of dataSource inside the expect or not? 
```

Comment:
Request for change: I would put this comment in a new Trello ticket.

Reply:
Adding ticket to the backlog and changing the comment to describe why we set the test up that way.